appveyor.yml

```diff
@@ -32,6 +33,7 @@ install:
     - IF %PHP%==1 cd ..
     - IF %PHP%==1 echo @php %%~dp0composer.phar %%* > composer.bat
     - IF %PHP%==1 copy /Y php.ini-development php.ini-min
+    - IF %PHP%==1 echo max_execution_time=1200 >> php.ini-min
```

Comment:
20 minutes is twice the time required for tests to run. This prevents blocking for appveyor queue for too long.